### ISSUE TITLE
progress counter for the compilation

### ISSUE THREAD

**Issue by wkotlarski** [FlexibleSUSY/FlexibleSUSY#104]
Would be great if we could show to the user how many files are left to compile. @Expander, what do you think? Is it even possible with the way how the makefile works?

**Comment by Expander** [comment]
Good idea! I've found this hint, which uses pure make and shell features:
https://stackoverflow.com/questions/451413/make-makefile-progress-indication/455390#455390
